server/config.js

```diff
@@ -6,7 +6,8 @@ const cache = new Map()
 const defaultConfig = {
   webpack: null,
   poweredByHeader: true,
-  distDir: '.next'
+  distDir: '.next',
+  assetPrefix: ''
```

Comment:
No any big reason. Do we really need it? Usually this is a build time config. I think it's better to  have this a just a string.